Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM sonarqube:7.4-community
```

Comment:
I did some local testing and apparently the plugin is not working properly with SonarQube versions 7.4 and 7.5. After the analysis with sonnar-scanner, the issues are reported fine but the project Overview reports: _This project is empty._
I tested against version 7.0 and 7.1 and everything seems fine.  I will open an issue to deal with this but for now I would say we should keep version 7.1.

Reply:
I see. Make sense. I just realised the previous versions were working because SonarQube was displaying the found issues even without the loc-statistics.
Yes, let's add that to the README.md.

Reply:
Downgraded to 7.1 with instructions for running in newer versions.